backend/BCAPIServer/src/main/java/daos/User.java

```diff
@@ -498,6 +502,27 @@ public List<String> allFriends() throws SQLException {
     return friends;
   }
 
+  /**
+   * List all the book clubs that this user is subscribed to.
+   *
+   * @return list of this user's book clubs
+   * @throws SQLException if something goes wrong with the database
+   */
+  public List<String> allClubs() throws SQLException {
+    List<String> clubs = new ArrayList<>();
+
+    getClubsStatement.clearParameters();
+    getClubsStatement.setString(1, this.user);
+
+    ResultSet rs = getClubsStatement.executeQuery();
+    while (rs.next()) {
+      clubs.add(rs.getString("book_key"));
```

Comment:
Would it be possible to return a list of book club objects and not just keys? This helps the load time in the frontend